### PR TITLE
Documentation: update docs, screenshots ahead of Paperless-ngx v2.0

### PR DESCRIPTION
**Pull request by shamoon**
## Proposed change

Welcome any feedback on the docs changes (or anything obv). Just to list out the changes:

- Reformatted absolute internal links so mkdocs doesnt complain (I do want to double-check this doesnt cause any issues), sorry this does make the PR a little less easy to review
    - Only real content changes (mostly minor stuff) are README.md, docs/index.md and docs/usage.md
- Updated screenshots
- Updated readme to be a little less verbose and link to docs for feature list
- Updated features list in docs (which required a change to prettier markdown formatting rules to allow nested list... for some reason)
- Added mkdocs Lightbox support
- Oh, and a footnote =)

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Other (please explain): Docs

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have included testing coverage for new code in this PR, for [backend](https://docs.paperless-ngx.com/development/#testing) and / or [front-end](https://docs.paperless-ngx.com/development/#testing-and-code-style) changes.
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [x] If applicable, I have checked that all tests pass, see [documentation](https://docs.paperless-ngx.com/development/#back-end-development).
- [x] I have run all `pre-commit` hooks, see [documentation](https://docs.paperless-ngx.com/development/#code-formatting-with-pre-commit-hooks).
- [x] I have made corresponding changes to the documentation as needed.
- [x] I have checked my modifications for any breaking changes.

### CODE REVIEW

**Comment by stumpylog** [comment]
I do like the cleaner documentation buidling.  I'll spin a build and give the changes a look there.  Agree it was high time to update the screenshots.

**Comment by stumpylog** [comment]
Depending if you want the images a little smaller, but lossly, pngquant might be a good option

**Comment by shamoon** [comment]
Yea I wasn't sure if it's too much, but I kinda hate pixel loss especially for things like this... I ran them through a program that tries PNGOUT, OxiPNG, AdvPNG and PNGCrush =)

**Comment by stumpylog** [comment]
Probably good enough. What do you think about the format I used for args to the new fuzzy match tool?  When I'm back I was thinking about standardizing all those

**Comment by shamoon** [comment]
Oh yea thats really nice

File: README.md [hunk 1]
```diff
@@ -16,8 +16,7 @@
 
 Paperless-ngx is a document management system that transforms your physical documents into a searchable online archive so you can keep, well, _less paper_.
 
-Paperless-ngx forked from [paperless-ng](https://github.com/jonaswinkler/paperless-ng) to continue the great work and distribute responsibility of supporting and advancing the project among a team of people. [Consider joining us!](#community-support) Discussion of this transition can be found in issues
-[#1599](https://github.com/jonaswinkler/paperless-ng/issues/1599) and [#1632](https://github.com/jonaswinkler/paperless-ng/issues/1632).
+Paperless-ngx is the official successor to the original 'Paperless' & 'Paperless-ng' projects and is designed distribute responsibility of supporting and advancing the project among a team of people. [Consider joining us!](#community-support).
```
Comment (tooomm) [inline README.md:19]: 👍 

Spotted some minor typos.
```suggestion
Paperless-ngx is the official successor to the original 'Paperless' & 'Paperless-ng' projects and is designed to distribute responsibility of supporting and advancing the project among a team of people. [Consider joining us!](#community-support)
```

The docs use the same wording:
https://github.com/paperless-ngx/paperless-ngx/pull/4542/files#diff-b4d68dc855d0f9476d3f2ee343853bd21bf82ea9960d0cf06661baa244439dd6R50

There, it actually also says "...projects and is designed **to continue the great work** and distribute responsibility..."

Comment (shamoon) [reply README.md]: Thanks, yea fair enough. I've unified the language.

Comment (tooomm) [reply README.md]: Readme looking good now! 👍 

(FYI, I'm lacking permissions to provide reviews on the repo 😉)

File: docs/configuration.md [hunk 2]
```diff
@@ -708,7 +708,7 @@ but could result in missing text content.
 additional arguments you wish to pass to OCRmyPDF. Since Paperless
 uses the API of OCRmyPDF, you have to specify these in a format that
 can be passed to the API. See [the API reference of
-OCRmyPDF](https://ocrmypdf.readthedocs.io/en/latest/api.html#reference)
+OCRmyPDF](https://ocrmypdf.readthedocs.io/en/latestapi.md.html#reference)
```
Comment (tooomm) [inline docs/configuration.md:711]:
```suggestion
OCRmyPDF](https://ocrmypdf.readthedocs.io/en/latest/api.html#reference)
```

**Review by stumpylog — APPROVED** [review]

**Pull request merged**